### DISCUSSION TITLE
Internal images not displaying when provider is set to 'ipx' (default)

### DISCUSSION THREAD

**Discussion by m-shum** [nuxt/image#1634] [Q&A]
**Issue Description**

Our build is deployed on Netlify, but our assets are external. Omitting .provider in Nuxt Image config automatically sets Netlify as the provider and breaks all external URLs even if the domain for those external assets is provided in the .domains array in Nuxt Image config. 

I have tried explicitly setting netlify as the provider and adding a baseURL, however, external images are still broken in deployment.

---

**Nuxt Image config:**
```
  image: {
    // Options
    formats: ['webp'],
    domains: ['xxx.xxx.com', 'xxx.xxx.com'],
    provider: 'ipx',
    quality: 80,
  },
  ```
  
**Setting .provider to 'ipx' (the default) fixes the external image issue, but breaks internal images even if they're housed inside the public directory without subfolders.**

How do we set up the default provider with external images? The documentation simply says to place all assets inside the public directory.

**Comment by Baroshem** [comment]
Hey,

What about setting a custom provider?
